src/common/signalk/signalk.json

```diff
@@ -36,6 +36,54 @@
       "units": "K",
       "description": "Current water temperature"
     },
+    {
+      "path": "environment.wind",
```

Comment:
This is not actually a value in SignalK. You cannot just provide `environment.wind` so we do not need this one.